### DISCUSSION TITLE
How can we incentivize people to join and actively contribute to our core working groups?

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [OBOFoundry/OBOFoundry.github.io#2624] [Polls]
Related to: https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2561

Problem: Many OFOC members; circa half of the named members join calls more or less regularly; few are active in working groups.

The editorial working groups need more people to stem the workload. Please respond to the poll below.

**Comment by nataled** [comment]
What does the third one ("Form a meeting preparation and ontology review committee...") mean?

**Comment by matentzn** [comment]
Not sure we need to vote on which specific issue to handle - all of these should be done imo.

I like the idea of relieving active members of WGs of review duties as a means to incentivise other to become more active; but for me, the think that has worked the best so far was 

1. creating clearly defined roles
2. hiring for these roles and creating SOPs
